src/lib/connectors/water/application-state.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const { serviceRequest } = require('@envage/water-abstraction-helpers');
+const urlJoin = require('url-join');
+
+const config = require('../../../../config');
+
+const getUrl = key => {
```

Comment:
Is it preferable for the import state to remain in the water service rather than move to this code module?